src/schema/v2/partner.ts

```diff
@@ -39,9 +44,55 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
   name: "Partner",
   interfaces: [NodeInterface],
   fields: () => {
+    const ArtistPartnerConnection = connectionWithCursorInfo({
```

Comment:
possibly worth adding a comment above that this avoids a circular require. The eventual error message we get from our circular/nested schema organization is usually non-obvious (where someone to move this out of the thunk).